tests/e2e/matmul/CMakeLists.txt

```diff
@@ -29,6 +29,31 @@ iree_generated_trace_runner_test(
     "--iree-opt-data-tiling=false"
   LABELS
 
+  TARGET_CPU_FEATURES_VARIANTS
+    "default"
+)
+
+iree_generated_trace_runner_test(
+  NAME
+    e2e_matmul_nondt_i8_i32_large
+  GENERATOR
+    "generate_e2e_matmul_tests.py"
+  GENERATOR_ARGS
+    "--lhs_rhs_type=i8"
+    "--acc_type=i32"
+    "--shapes=large"
+  TRACE_RUNNER
+    iree-e2e-matmul-test
+  TARGET_BACKENDS
+    "llvm-cpu"
+  DRIVERS
+    "local-task"
+  COMPILER_FLAGS
+    "--iree-opt-data-tiling=false"
+  LABELS
+
+  TARGET_CPU_FEATURES_VARIANTS
+    "default"
 )
```

Comment:
This change is unrelated to SVE, right?

Reply:
I wanted to update the existing LLVMCPU tests to cover large/small matmuls with SVE. The widening tests are not supported, so currently SVE is not enabled there. Note this CMake file is generated from the BUILD.bazel file.